setting_up_firely_server/configuration/environment_variables.rst

```diff
@@ -139,6 +139,19 @@ When the 'ASPNETCORE_ENVIRONMENT' variable is set to development the call stack
    
    ASPNETCORE_ENVIRONMENT=Development
 
+.. _customize_config_location:
```

Comment:
This documentation would get the job done but is not on par with the rest. It has several typos (I recommend [Grammarly](https://www.grammarly.com/), even the free version is fine). And it is imho too concise.
Since I also thought it would be better placed in `appsettings.rst`, I took the liberty of pushing a new commit. Let me know what you think.